### DISCUSSION TITLE
Moving Focus in the Bookmark Menu

### DISCUSSION THREAD

**Discussion by BonfireAtNight** [ahrm/sioyek#399] [Ideas]
I'm using the bookmark functionality for writing short summaries of sections and paragraphs. For this it's great that the bookmark closest to the current position in the document is highlighted (I love that!). However, as you collect more bookmarks (as you surely will in a book file), the highlighted bookmarks move downwards and eventually are no longer visible when you open the bookmark menu. 

If you scroll down, it's still properly highlighted, but it would be much more convenient if the default focus (the area of bookmarks you see when opening the goto_bookmark menu) moved along.

**Comment by ahrm** [comment]
Should be fixed in e8eddc0d3688e0bbd6b0d1fae7fad6a0d6d01726.

**Reply by BonfireAtNight** [reply]
Great, looking forward to that. Thanks!
